components/game/block-preview.js

```diff
@@ -42,6 +46,13 @@ class PreviewBlockManager {
       this.previewBlockArray[i].draw(ctx);
     }
   }
+
+  _drawTitle(ctx) {
```

Comment:
This seems repeated with ShiftBlock's. Perhaps we could make an abstraction between PreviewBlockManager and ShiftBlock.

I guess this function could be generalized for drawing text on the canvas. 
So I suggest that moving out of the classes to another location(like a utility? or global?) with renaming as `drawLabel` then receive parameters that are required information such as canvas-context, position, text, and so on.

Example:
```javascript
 drawLabel(ctx, 'NEXT', shiftBackgroundColor, '16px', previewBlockLocation[0][0], 15)
```

I personally think handling the context should be wrapped by a function for readability.

WDYT about this?

Reply:
Agree.

I think it would be very good to refactoring it like that.